frontend/src/App.tsx

```diff
@@ -17,6 +18,7 @@ function App() {
         <Hamburger />
         <Routes>
           <Route path="/" element={<Home />} />
+          <Route path="/submit-sighting" element={<SubmitSighting />} />
```

Comment:
I think Robert and Ruksan are working on the ticket to implement this, it would be a good idea to check the naming and location they are implementing so you can make sure this matches what they're doing. It will make merging things easier later on